discovery-provider/src/queries/queries.py

```diff
@@ -111,6 +111,13 @@ def get_tracks():
                 Track.owner_id == user_id
             )
 
+        # Allow filtering of deleted tracks
+        if "filter_deleted" in request.args:
```

Comment:
Potential future bug: if we pass filter_deleted = false as a query param, we're going to be now start filtering deleted out b/c we just test for it's existence and bc we don't test for truthy down below when adding the param

Reply:
I updated this to be more explicit to only with the string value of `true`. But didn't add functionality to filter tracks the other way (Returning only deleted tracks).